### PR TITLE
GUI styling update 

### PR DESCRIPTION
**Pull request by scottaames**

### CODE REVIEW

File: app/src/components/presentational/ConsoleCoordinates.jsx [hunk 1]
```diff
@@ -17,27 +17,29 @@ const useStyles = makeStyles({
   },
   title: {
     color: "#343a40",
-    fontSize: "0.7rem",
+    //fontSize: "12px",
```
Comment (kaitlyndlee) [inline app/src/components/presentational/ConsoleCoordinates.jsx]: If this is commented, can it be removed?

**Comment by kaitlyndlee** [comment]
When I tried pulling down the changes and running npm start, it gave me this error:
`Error: Cannot find module '../../assets/img/cartocosmos-logo.svg'`

**Comment by kaitlyndlee** [comment]
Minor thing, the lat/lon buttons seem too off-centered. Is it possible to move them over to the right just a bit? Maybe so that there is equal spacing between the buttons and lat/lon coords, and the lat/lon coords and the right of the GUI?

**Comment by scottaames** [comment]
> Minor thing, the lat/lon buttons seem too off-centered. Is it possible to move them over to the right just a bit? Maybe so that there is equal spacing between the buttons and lat/lon coords, and the lat/lon coords and the right of the GUI?

Should be fixed now.

**Review by kaitlyndlee — APPROVED** [review]
Looks good!

**Review by jkaufy — APPROVED** [review]
Looks good to me

**Review by cgm78 — APPROVED** [review]

**Pull request merged**